v2/uuid/pgxcompat/README.md

```diff
@@ -0,0 +1,15 @@
+This package enables PGX to use parameters of type go-utility/v2/uuid. To enable support, register the type in the `AfterConnect` callback before connecting to the database.
+
+```golang
+dbconfig, err := pgxpool.ParseConfig(databaseURL)
+if err != nil {
+	// handle error
+}
+dbconfig.AfterConnect = func(ctx context.Context, conn *pgx.Conn) error {
+	conn.ConnInfo().RegisterDataType(pgtype.DataType{
+		Value: &pgxcompat.UUID{},
+		Name:  "uuid",
+		OID:   pgtype.UUIDOID,
+	})
+	return nil
+}```
```

Comment:
Does it make sense for this code to live where it lives?
Can you import v2/uuid without also importing its subpackages?
One wouldn't want "database/sql/driver" and "github.com/jackc/pgtype" for instance when only interested in the uuid package

Reply:
I'm not sure, conceptually I like having it under uuid, but maybe it should be next to it instead.

Reply:
Moved to v2/pgxcompat